src/components/SignupForm.jsx

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable react/require-default-props */
```

Comment:
#### Critical
- Don't do this.

Reply:
Alright. I have taken it out. https://github.com/andela/apollo-ah-frontend/blob/feature%2F164198142%2Fsignup/src%2Fcomponents%2FSignupForm.jsx